README.md

```diff
@@ -190,6 +190,7 @@ The following variables are added to the context by all of the above tags, which
 - **`text`**: The text that should be used for the menu item
 - **`active_class`**: A class name to indicate the 'active' state of the menu item. The value will be 'active' if linking to the current page, or 'ancestor' if linking to one of it's ancestors.
 - **`has_children_in_menu`**: A boolean indicating whether the menu item has children that should be output as a sub-menu.
+- **`handle`**: A string which can be used to do specific matching of menu items in the template.
```

Comment:
Hi @tleguijt. `menu_items` is sometimes a list of `MenuItem` objects with `text`, `href` and `active_class` attributes added (e.g. for a `flat_menu`, or the top-level items in a `main_menu`), but more often, it is a list of `Page` objects with those same attributes added (If rendering a `sub_menu`, `section_menu` or `children_menu`).

Because of this, saying that `handle` is always present is a little misleading, because it's only present in some cases and not others. I'm thinking it might be better to document this feature in the instructions for setting up flat and main menus in the CMS instead? That along with some better help text would do the trick, i think.